devops-970.nix

```diff
@@ -0,0 +1,144 @@
+{ cluster ? "staging", signingKeys ? null, fudgeConfig ? null }:
+
+let
+  old = import ./. { inherit cluster; };
+  inherit (old) pkgs;
+  inherit (pkgs) lib;
+  newpkgssrc = pkgs.fetchFromGitHub {
+    owner = "nixos";
+    repo = "nixpkgs";
+    rev = "be445a9074f";
+    sha256 = "15dc7gdspimavcwyw9nif4s59v79gk18rwsafylffs9m1ld2dxwa";
+  };
+  newpkgs = import newpkgssrc {
+    #crossSystem = (import "${newpkgssrc}/lib").systems.examples.mingwW64;
+    #localSystem.system = "x86_64-linux";
+  };
+  ghcWithCardano = old.temp.haskellPackages.ghcWithPackages (ps: [ ps.cardano-sl ps.cardano-sl-x509 ]);
+  packages = self: {
+    inherit old;
+    dlls = pkgs.fetchurl {
+      url = "https://s3.eu-central-1.amazonaws.com/daedalus-ci-binaries/DLLs.zip";
+      sha256 = "0p6nrf8sg2wgcaf3b1qkbb98bz2dimb7lnshsa93xnmia9m2vsxa";
+    };
+    configMutator = pkgs.runCommand "configMutator" { buildInputs = [ ghcWithCardano ]; } ''
+      cp ${./ConfigMutator.hs} ConfigMutator.hs
+      mkdir -p $out/bin/
+      ghc ConfigMutator.hs -o $out/bin/config-mutator
+    '';
+    nsis = newpkgs.callPackage ./nsis.nix {};
+    cardanoJSON = builtins.fromJSON (builtins.readFile ./cardano-sl-src.json);
+    cardanoSrc = pkgs.fetchFromGitHub {
+      owner = "input-output-hk";
+      repo = "cardano-sl";
+      rev = self.cardanoJSON.rev;
+      sha256 = self.cardanoJSON.sha256;
+    };
+    cross = (import (self.cardanoSrc + "/release.nix") { cardano = { outPath = self.cardanoSrc; rev = self.cardanoJSON.rev; }; }).daedalus-mingw32-pkg;
```

Comment:
This attribute name is confusingly generic -- I'd try adding more enlightenment to it..